english/love-udacity/love-udacity.md

```diff
@@ -4,4 +4,4 @@ Getting books and buying courses didn't help until I met Udacity.
 
 She has been the difference and now I am confident I can be a developer.
 
-I am glad you also met [Udacity](http://www.udacity.com)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.